lms/djangoapps/courseware/views/views.py

```diff
@@ -1644,15 +1646,23 @@ def render_public_video_xblock(request, usage_key_string):
             will_recheck_access=False
         )
 
-        # video must be public (`Public Access` field set to True) by course author in studio in video advanced settings
-        if not block.public_access:
-            raise Http404("Video not found.")
+        fragment = block.render(view, context={
+            'public_video_embed': is_embed_view,
+        })
 
-        fragment = block.render(view, context={})
+        video_description = f"Watch a video from the course {course.display_name} "
+        if course.display_organization is not None:
+            video_description += f"by {course.display_organization} "
+        video_description += "on edX.org"
 
         context = {
             'fragment': fragment,
             'course': course,
+            'video_title': block.display_name_with_default,
+            'video_description': video_description,
+            'video_thumbnail': "https://www.edx.org/images/logos/edx-logo-elm.svg",
```

Comment:
This link to the edX logo does not work, I think it has something to do with robots.txt
The commented out link to a youtube thumbnail DOES work. I want to mess with our thumbnail / poster stuff a little more.